src/main/java/TaskList.java

```diff
@@ -0,0 +1,135 @@
+import java.util.ArrayList;
+
+/**
+ * TaskList class Used to store an ArrayList of Tasks Handles the various type of commands for
+ * modifying the list
+ */
+public class TaskList {
+  private final ArrayList<Task> tasks;
+
+  /**
+   * Constructor to wrap an ArrayList<Task>
+   *
+   * @param tasks ArrayList of Tasks
+   */
+  public TaskList(ArrayList<Task> tasks) {
+    this.tasks = tasks;
+  }
+
+  /**
+   * Handles the addition of an event
+   *
+   * @param inputs Userinput split into tokens
+   */
+  public String handleEvent(String[] inputs) {
+    String[] tokens = inputs[1].split(" /at ");
+    tasks.add(new Event(tokens[0], tokens[1]));
+    System.out.println("Got it. I've added this task:");
+    System.out.println(tasks.get(tasks.size() - 1));
+    System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+    return "Got it. I've added this task:\n" +  tasks.get(tasks.size() - 1) + "\nNow you have "
```

Comment:
I noticed this is duplicated in quite a few methods. Should this be extracted out?